packages/@aws-cdk/aws-iotevents/test/input.test.ts

```diff
@@ -39,9 +42,41 @@ test('can get input name', () => {
   });
 });
 
-test('can set physical name', () => {
-  const stack = new cdk.Stack();
+test('can get input ARN', () => {
```

Comment:
Can we move this new test below `'can set physical name'`? The diff will be nicer that way.

Reply:
I've fix this, but the diff maybe not changed to expected. Because I refactor in this PR, this  the diff became confusable I think😓. Sorry.

Reply:
It's fine, don't worry about it 🙂.